pkg/builder/nop/builder_test.go

```diff
@@ -137,3 +139,20 @@ func TestOperationFromStatus(t *testing.T) {
 		t.Errorf("ErrorMessage(%v); wanted not failed, got %q", status, msg)
 	}
 }
+
+func TestTerminate(t *testing.T) {
+	builder := Builder{
+		OpErr: errors.New("testerr"),
+	}
+	op, err := builder.OperationFromStatus(&v1alpha1.BuildStatus{
+		Google: &v1alpha1.GoogleSpec{
```

Comment:
Might be out of the scope of this PR, but we should rename GoogleSpec since we don't have google builder anymore

Reply:
@ImJasonH mentioned in earlier comment that about cleaning this up in v1alpha2

Reply:
The only thing that uses `GoogleSpec` anymore is the nop builder, and I think as an aspiration I'd like it'd be cool to see if we can get rid of nop builder too, and just use a cluster builder implementation backed by fake k8s clients.

But yeah, as for dropping `GoogleSpec` I think we have to wait until v1alpha2 to make breaking changes, and CRD versioning won't have a good story for a couple more k8s releases it looks like :(